src/needs/need.model.ts

```diff
@@ -23,6 +23,10 @@ const NeedSchema = new Schema({
         type: Number,
         require: true
     },
+    fulfilled: {
+        type: Boolean,
+        // require: true
+    }
```

Comment:
this should not be a boolean, it should be a reference to the `Fulfillment` data model. Similar to how this model references a User, so too do we need this `fulfilled` field to reference a `Fulfillment` model